packages/sdk/src/evm/index.ts

```diff
@@ -50,3 +50,6 @@ export type { SmartContract } from "./contracts/smart-contract";
 
 // re-export from functions entry point
 export * from "./functions";
+
+// export zk-sync specific functions
+export * from "./zksync";
```

Comment:
dont export in the main /evm - should be available only if ppl import /evm/zksync specifically